charts/images.yaml

```diff
@@ -154,6 +160,29 @@ images:
   repository: alpine
   tag: "3.8"
 
+# CSI
+- name: csi-attacher
+  sourceRepository: https://github.com/kubernetes-csi/external-attacher
+  repository: quay.io/k8scsi/csi-attacher
+  tag: v0.3.0
+  version: 1.11.x
```

Comment:
why is this needed? does that mean that we don't support alicloud cluster's `< 1.11` ?

Reply:
Alright, thanks!